src/components/ReplayableVideoAsset.vue

```diff
@@ -26,6 +28,7 @@
       v-if="!showsControls"
       class="control-button"
       href="#"
+      :aria-describedby="id || null"
```

Comment:
Did you mean to do something like `alt ? id : null` or `alt && id` here since we only add the description element with the id if there is alt text?

Reply:
We only pass the id if[ there is an alt property at the Asset component](https://github.com/apple/swift-docc-render/pull/784/files#diff-9267d8f28e02bad68c9715386d639fefd148b7ef49489bb469b299fe1deccad3R114), but you are right, adding `|| null` is useless here, Vue won't populate the attribute if it's an undefined value anyways.

Reply:
https://github.com/apple/swift-docc-render/pull/784/commits/8ffbb24898c76e4007589f624ea79a5639f0c3a9